### DISCUSSION TITLE
Elsa 3.1  - Fluent workflow builder

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5250] [Q&A]
Is Fluent builder for workflow not supported with Elsa 3.1 ?

**Comment by glime-ai** [comment]
Based on the provided context, there is no direct mention of the Fluent builder's support status in Elsa 3.1. However, the context does indicate that Elsa is continually evolving and that certain features and activities are still in development or testing phases. It's possible that the Fluent builder, if it is a feature or part of the Elsa workflow ecosystem, may also be under development or not fully documented yet.

To get a definitive answer on the support status of the Fluent builder in Elsa 3.1, I would recommend checking the official Elsa documentation, the GitHub repository for any updates or release notes, or reaching out to the Elsa community or support channels for clarification.

Would you like me to provide guidance on where to find this information or assist you with anything else related to Elsa workflows?

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
